models/util/convert-data-mixin.js

```diff
@@ -24,7 +24,12 @@ module.exports = {
         }
       }
 
-      data = Type.data.wrap(data, errors)
+      if (Type.data.strict === false) {
+        data = Type.data.wrap(data)
```

Comment:
Won't this completely hide errors? Even if we're trying to be non-strict for future GH changes, we should probably at least log something to let us know it's happening.